### PR TITLE
Update README spec compliance section

### PR DESCRIPTION
**Pull request by tyiu**
This update reflects the NIPs repo at this SHA:
https://github.com/nostr-protocol/nips/blob/1a106c6bff20a9408fe0ec51b265e3c5de86d25f/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,34 +23,32 @@ Nostr SDK iOS implements the following NIPs:
 - [ ] [NIP-09: Event Deletion](https://github.com/nostr-protocol/nips/blob/master/09.md)
 - [x] [NIP-10: Conventions for clients' use of `e` and `p` tags in text events](https://github.com/nostr-protocol/nips/blob/master/10.md)
 - [x] [NIP-11: Relay Information Document](https://github.com/nostr-protocol/nips/blob/master/11.md)
-- [ ] [NIP-12: Generic Tag Queries](https://github.com/nostr-protocol/nips/blob/master/12.md)
 - [ ] [NIP-13: Proof of Work](https://github.com/nostr-protocol/nips/blob/master/13.md)
 - [ ] [NIP-14: Subject tag in text events](https://github.com/nostr-protocol/nips/blob/master/14.md)
 - [ ] [NIP-15: Nostr Marketplace (for resilient marketplaces)](https://github.com/nostr-protocol/nips/blob/master/15.md)
-- [ ] [NIP-16: Event Treatment](https://github.com/nostr-protocol/nips/blob/master/16.md)
```
Comment (tyiu) [inline README.md:30]: Moved to NIP-01.

File: README.md [hunk 2]
```diff
@@ -23,34 +23,32 @@ Nostr SDK iOS implements the following NIPs:
...
 - [x] [NIP-18: Reposts](https://github.com/nostr-protocol/nips/blob/master/18.md)
 - [ ] [NIP-19: bech32-encoded entities](https://github.com/nostr-protocol/nips/blob/master/19.md)
-- [ ] [NIP-20: Command Results](https://github.com/nostr-protocol/nips/blob/master/20.md)
```
Comment (tyiu) [inline README.md:33]: Moved to NIP-01.

File: README.md [hunk 3]
```diff
@@ -23,34 +23,32 @@ Nostr SDK iOS implements the following NIPs:
...
 - [ ] [NIP-23: Long-form Content](https://github.com/nostr-protocol/nips/blob/master/23.md)
+- [x] [NIP-24: Extra metadata fields and tags](https://github.com/nostr-protocol/nips/blob/master/24.md)
 - [x] [NIP-25: Reactions](https://github.com/nostr-protocol/nips/blob/master/25.md)
 - [ ] [NIP-26: Delegated Event Signing](https://github.com/nostr-protocol/nips/blob/master/26.md)
 - [ ] [NIP-27: Text Note References](https://github.com/nostr-protocol/nips/blob/master/27.md)
 - [ ] [NIP-28: Public Chat](https://github.com/nostr-protocol/nips/blob/master/28.md)
 - [x] [NIP-30: Custom Emoji](https://github.com/nostr-protocol/nips/blob/master/30.md)
 - [ ] [NIP-31: Dealing with Unknown Events](https://github.com/nostr-protocol/nips/blob/master/31.md)
 - [ ] [NIP-32: Labeling](https://github.com/nostr-protocol/nips/blob/master/32.md)
-- [ ] [NIP-33: Parameterized Replaceable Events](https://github.com/nostr-protocol/nips/blob/master/33.md)
```
Comment (tyiu) [inline README.md:44]: Moved to NIP-01.

File: README.md [hunk 4]
```diff
@@ -23,34 +23,32 @@ Nostr SDK iOS implements the following NIPs:
...
+- [ ] [NIP-38: User Statuses](https://github.com/nostr-protocol/nips/blob/master/38.md)
 - [ ] [NIP-39: External Identities in Profiles](https://github.com/nostr-protocol/nips/blob/master/39.md)
 - [ ] [NIP-40: Expiration Timestamp](https://github.com/nostr-protocol/nips/blob/master/40.md)
 - [ ] [NIP-42: Authentication of clients to relays](https://github.com/nostr-protocol/nips/blob/master/42.md)
 - [ ] [NIP-45: Counting results](https://github.com/nostr-protocol/nips/blob/master/45.md)
 - [ ] [NIP-46: Nostr Connect](https://github.com/nostr-protocol/nips/blob/master/46.md)
 - [ ] [NIP-47: Wallet Connect](https://github.com/nostr-protocol/nips/blob/master/47.md)
 - [ ] [NIP-48: Proxy Tags](https://github.com/nostr-protocol/nips/blob/master/48.md)
-- [ ] [NIP-50: Keywords filter](https://github.com/nostr-protocol/nips/blob/master/50.md)
+- [ ] [NIP-50: Search Capability](https://github.com/nostr-protocol/nips/blob/master/50.md)
```
Comment (tyiu) [inline README.md:51]: Renamed from Keywords filter to Search Capability.

**Review by bryanmontz — APPROVED** [review]

**Pull request merged**